### PR TITLE
Fix null last_price, add combos enums and combo name extractors

### PR DESCRIPTION
**Pull request by Tradyn**
Hi, hopefully this PR is up to date and correct.

Added Combos / types and other helpers to tickers / instruments.

Thanks for your hard work on this.

### CODE REVIEW

File: src/DeriSock/Model/Objects/Instrument.cs [hunk 1]
```diff
@@ -10,6 +13,11 @@ public partial class Instrument
   [JsonIgnore]
   public InstrumentType InstrumentType => InstrumentName.GetInstrumentType();
 
+  ///   Get combo type
```
Comment (psollberger) [inline src/DeriSock/Model/Objects/Instrument.cs:16]:
```suggestion
  /// <summary>
  ///   Get combo type
```

**Review by psollberger — CHANGES_REQUESTED** [review]
Thank you so much for your contribution. It's really motivating :)

I proposed a change in Instrument.cs. A xml-doc comment isn't correct. Can you address that?

Also keep in mind that the files ending in `.g.cs` are generated files and require adjustments in DevTools project.
I did thos adjustments and already pushed them, so this is ok now.

Can you please tell me which request produces the null values in Instrument?
Especially the `instrument_id` does not make sense to me. That should never be null, shouldn't it?

**Comment by Tradyn** [comment]
The last_price is the problem value.
That can be null.

instrument_id .... I have no idea??? It must be a mistake I don't have any issues with that...??

**Comment by Tradyn** [comment]
Fixed summary comment.
Changed instrument_id? to non nullable.
Mistake on that one.
I haven’t started on orders or books yet.. 🫣

**Review by psollberger — APPROVED** [review]

**Pull request merged**